step_subscribe_cancellation.go

```diff
@@ -25,6 +35,11 @@ func (s *stepSubscribeCancellation) Cleanup(state multistep.StateBag) {
 		return
 	}
 
+	ctx := state.Get("ctx").(gocontext.Context)
+
+	ctx, span := trace.StartSpan(ctx, "SubscribeCancellation_cleanup")
```

Comment:
Perhaps we can name these `SubscribeCancellation.Run` and `SubscribeCancellation.Cleanup` respectively? (that would include adjusting everything else to follow that naming convention)